### ISSUE TITLE
Some nodes are missing from result edges

### ISSUE THREAD

**Issue by maximusunc** [ranking-agent/strider#442]
PK in CI: 067ed81a-dada-4334-8140-4792c9c2d1d9

**Comment by cbizon** [comment]
I (accidentally) ran with cache-override on, and it produced the same problem.  It seems that there is one node missing: PUBCHEM.COMPOUND:12198448, which is referenced in 3 edges (all support edges somewhere I think)

**Comment by maximusunc** [comment]
Update: I believe I've tracked this down to a kp giving back a subclass result without a corresponding query id. We try and handle that and just assign it a valid query id, but I think that is causing this issue. Strider ends up returning an edge binding in the results that references the subclass node, but nothing else references it. Then Aragorn scores/sorts/filters, and the kgraph orphan filter then filters that node out, causing the TRAPI validation error.

I think we should probably just throw out any subclass results without a valid query id. In this case, there was only 1. I had only added this temporary query id fix because the automats weren't returning it correctly and didn't want to throw them all out.
